configure.ac

```
@@ -5,7 +5,8 @@ dnl Process this file with autoconf to produce a configure script.
 dnl You may need to use autoheader as well if changing any DEFINEs
 
 AC_PREREQ([2.69])
-AC_INIT([Darshan],[3.4.2],[darshan-users@lists.mcs.anl.gov],
+AC_INIT([Darshan], m4_esyscmd_s([cat ./darshan.version]),
```

```suggestion
AC_INIT([Darshan], m4_normalize(m4_include([./darshan.version])),
```

Avoids running /usr/bin/cat.

---

Updated to use this better solution. Thanks.